app/src/main/java/com/smashingboxes/epa_prototype_android/MainActivity.java

```diff
@@ -178,7 +185,22 @@ protected void onCreate(Bundle savedInstanceState) {
         epaRequestManager = new EpaRequestManager(this, this);
 
         initializeSwipeHandling();
-        fetchData();
+
+        if(savedInstanceState != null && savedInstanceState.containsKey(KEY_EXTRA_AIR_QUALITY_LIST) && savedInstanceState.containsKey(KEY_EXTRA_TIME_SERIES_LIST)) {
```

Comment:
The likelyhood of that happening is really low, so it's ignored presently.  The only way that could happen is if onStop() is called before both network calls finish, and that's partially negated by the onRestart() check.  I realize there is potential for it to cause an issue, but it's very unlikely, so it sufficient for prototype purposes.